index.html

```diff
@@ -9,6 +9,16 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 
     <title>Bellshade - Learn with Our Community</title>
+
+    <meta name="description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia" />
+    <meta property="og:locale" content="id_ID" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content='Bellshade - Learn with Our Community' />
+    <meta property="og:description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia" />
+    <meta property="og:image" content='https://cdn.statically.io/img/media.discordapp.net/w=800/attachments/891222600422936616/892682527720996884/unknown.png' />
+    <meta property="og:image:width" content="800" />
+    <meta property="og:image:height" content="800" />
```

Comment:
Kalo dari gambar yang cdn staticcally, ukurannya 800x856

```suggestion
    <meta property="og:image:height" content="856" />
```